CHANGES.rst

```diff
@@ -5,7 +5,7 @@
 4.7.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Switch package to src-layout.
```

Comment:
I wonder if this needs a change note? It's not visible to end users. Even other packagers shouldn't care unless they were doing unspeakable things (i.e., not using 'pip wheel' or 'python setup.py bdist_wheel'). 

Not that adding the change note hurts, I just worry about confusing end users ("Do I need to upgrade? I don't even know what that means.") Maybe a sentence to the effect that "this is a packaging only change" and a reference to the PR or issue would help.

Reply:
I incorporated your feedback into https://github.com/zopefoundation/persistent/pull/168/commits/ea09894c0b3ee6774459c950c68eb42849103102.